src/components/sbb-clock/sbb-clock.stories.js

```diff
@@ -2,77 +2,35 @@ import { h } from 'jsx-dom';
 import isChromatic from 'chromatic/isChromatic';
 import readme from './readme.md';
 
-const Template = (args) => <sbb-clock {...args} />;
-
-const times = {
+const dataNow = {
   control: {
-    type: 'select',
+    type: 'number',
```

Comment:
I think you can use the same approach as in other components where you can chose a date (date control or similar)